common/api/appui-react.api.md

```diff
@@ -3299,6 +3299,11 @@ export interface OpenChildWindowInfo {
 export class OpenMessageCenterEvent extends UiEvent<{}> {
 }
 
+// @beta
+export interface OverflowProps {
```

Comment:
I'd prefer to rename this to `ToolbarOverflowProps` to avoid a potential name clash in the future.

Reply:
Renamed to `OverflowToolbarOptions` to follow `CanFloatWidgetOptions` pattern.